rustls/tests/api.rs

```diff
@@ -272,6 +272,36 @@ fn config_builder_for_client_rejects_incompatible_cipher_suites() {
     );
 }
 
+#[cfg(feature = "tls12")]
```

Comment:
I think it would probably make sense to keep these tests in a separate file in `tests`, which can have an inner attribute for the `tls12` guard. That file can then inline the contents of `tests/common/ffdhe.rs` (at the bottom of the file).